src/main/java/io/github/jasmin/dao/UserDao.java

```diff
@@ -0,0 +1,51 @@
+package io.github.jasmin.dao;
+
+import io.github.jasmin.model.User;
+import org.springframework.stereotype.Repository;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Repository
+public class UserDao {
+    public static List<User> users;
+
+    static {
+        users = new ArrayList<>();
+        users.add(new User(1,"hyunsoo","mhs","1234"));
+        users.add(new User(2,"jungtae","pjt","1234"));
+        users.add(new User(3,"seongjae","lsj","1234"));
+        users.add(new User(4,"sangbum","lsb","1234"));
+        users.add(new User(5,"areum","jar","1234"));
+    }
```

Comment:
```suggestion
    private static Map<String, User> users = new HashMap<>();
```

`public`보다는 `private`는 어땠을까 궁금.

내부 자료구조로 db를 표현했다면 `Map`으로 표현하면 어땠을까 싶네요